operator/config/rbac/role.yaml

```diff
@@ -39,6 +39,12 @@ rules:
   verbs:
   - get
   - update
+- apiGroups:
+  - config.openshift.io
+  resources:
+  - clusterversions
+  verbs:
+  - '*'
```

Comment:
I think only get verb should be sufficient here ?

Reply:
We can handle this comment separately, at all other places for verbs we have "*"

Reply:
@madhuthorat @nitishkumar4 did we open the issue/PR for the same ?

Reply:
Being tracked at https://github.com/IBM/ibm-spectrum-scale-csi/issues/662